railties/Rakefile

```diff
@@ -52,7 +52,6 @@ namespace :test do
     end
     $-w = true
 
-    require "bundler/setup" unless defined?(Bundler)
```

Comment:
This removes a real line of code that gets executed (notably, before the following require, which might otherwise load the wrong `active_support`), and replaces it with an entry in an array named `fake_command` (though admittedly, not actually fake when run on JRuby/Windows).

Reply:
Thanks @matthewd!

I restored the line. I don't think it makes a difference for the `require` you mentioned because the `$LOAD_PATH` is getting setup right before to load the right version of `active_support`, but I see how it could affect requiring the right version of other dependencies when not running tests with `bundler`.

Regarding the `fake_command` array, yeah, that's the point of this PR, that you can copy-paste the fake command, run it, and it works ok, even if it's not really run by default.